### PR TITLE
framework/task_manager: Add checking handler logic for unicast

### PR DESCRIPTION
**Pull request by jeongchanKim**
If handler is not registered, unicast cannot work properly.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [14a986460a24169a66362b8c57790dec06c44f1b] -

**Comment by seinfra** [comment]
Target : [14a986460a24169a66362b8c57790dec06c44f1b] -

**Comment by seinfra** [comment]
Target : [c3ed6f5c0e7c418791e0bdbfded3d80deb94afd2] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [c3ed6f5c0e7c418791e0bdbfded3d80deb94afd2] - Code Rule Check OK.

File: os/kernel/signal/sig_checkhandler.c [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
```
Comment (sunghan-chang) [inline os/kernel/signal/sig_checkhandler.c]: Unncessary header

File: os/drivers/task_manager/task_manager_drv.c [hunk 2]
```diff
@@ -159,6 +159,7 @@ static ssize_t taskmgt_write(FAR struct file *filep, FAR const char *buffer, siz
 static int taskmgt_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 {
 	int ret = -EINVAL;
+	bool ret_handler = false;
```
Comment (sunghan-chang) [inline os/drivers/task_manager/task_manager_drv.c]: The ```ret``` is enough to use with typecasting.

File: os/kernel/signal/sig_checkhandler.c [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+#include <signal.h>
```
Comment (sunghan-chang) [inline os/kernel/signal/sig_checkhandler.c]: <signal.h> and <tinyara/signal.h> either.

**Comment by seinfra** [comment]
Target : [84748dae4167389a756b29c354138aee4e8f51c2] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [84748dae4167389a756b29c354138aee4e8f51c2] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [4008843148bb7677e4d448e44fe7d6bde0b4d22d] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [4008843148bb7677e4d448e44fe7d6bde0b4d22d] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [52061ddb3a2077e675db81b0aa5063d815f4b624] - Code Rule Check OK.

**Pull request merged**